cityofaustin/atd-moped
# Use REST connector to update moped project activity log events

## Associated issues
- https://github.com/cityofaustin/atd-data-tech/issues/10600

evaluate REST connector to update activity log when changes are done to the moped_project table
Also https://github.com/cityofaustin/atd-data-tech/issues/10811, tracking changes to moped_proj_tags

This is the companion PR for changes to the moped-etl code, to get the test environment deployed with the new environment variable. https://github.com/cityofaustin/atd-moped/pull/887/

## Testing
**URL to test:** 
https://10600-rest-activity--atd-moped-main.netlify.app/moped/projects/227?tab=summary


**Steps to test:**
Update something from the moped project such as the description, or ecapris ID or website. 
Make changes to the project tags. 

Then check the activity log. You may need to refresh the activity log to see any changes. I see the same behavior in staging right now, updates dont immediately show on the activity log page. I think the query isn't being refetched between tab changes. Let me know if fixing that is a blocker for this PR to be approved. 


---
#### Ship list
